masonite/drivers/CacheDiskDriver.py

```diff
@@ -0,0 +1,70 @@
+import glob
+import os
+import time
+
+
+class CacheDiskDriver():
```

Comment:
Ok since this is a new class we'll need some tests for it. Just make some basic tests that test all the methods here and put it inside `test_cache.py`